_build/css/main.css

```diff
@@ -457,3 +477,29 @@ table>tbody+tbody{
 .sandbox-about-title #sandbox-planet-logo {
   height: 44px;
 }
+
+.explore-sbd-btn {
```

Comment:
This would be better if it was renamed to `explore-psd-btn` as in planet-sandbox-data, also change it in the `.hbs` file appropriately

Reply:
Updated